content/data/scripts/briefing.lua

```diff
@@ -203,7 +203,7 @@ function BriefingController:createGoalItem(title)
 	goal_el:AppendChild(self:makeBullet())
 	
 	local goal_text = self.document:CreateElement("div")
-	goal_text.inner_rml = goal.Message .. "<br></br>"
+	goal_text.inner_rml = goal_el.Message .. "<br></br>"
```

Comment:
so I think this one actually needs to be

`goal_text.inner_rml = title .. "<br></br>"`

because the calling places send `goal.Message` as the argument. Originally they must have just sent the goal object.